yarn.lock

```diff
@@ -656,9 +656,9 @@
   integrity sha512-tHq6qdbT9U1IRSGf14CL0pUlULksvY9OZ+5eEgl1N7t+OA3tGvNpxJCzuKQlsNgCVwbAs670L1vcVQi8j9HjnA==
 
 "@types/node@*":
-  version "10.14.0"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.14.0.tgz#1c297530428c6f4e0a0a3222f5b44745669aa9f7"
-  integrity sha512-1UhSMMDix7bVdUeqtZERQQyJr3QuFoN5X5APtpIooGkumE3crPaeq7UgFeJNjGD8yCQ8od8PzRkgptR5+x327Q==
+  version "12.20.7"
```

Comment:
needed up to date (for Node 12) types to be able to use `stream.Readable.from()`. 

(@adamraine these outdated node types were also the thing preventing this suggestion from working: https://github.com/GoogleChrome/lighthouse/pull/12145#discussion_r598014045)